metadata.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: prometheus-edge-hub
+description: |
+  Prometheus Edge Hub is a replacement of the Prometheus Pushgateway which allows for the pushing of
+  metrics to an endpoint for scraping by prometheus, rather than having Prometheus scrape the metric
+  sources directly. This differs from the Prometheus Pushgateway in several ways, the most important
+  of which being that it does not overwrite timestamps and that metrics do not persist until updated.
+  When the hub is scraped, all metrics are drained.
+summary: |
+  Prometheus Edge Hub is a replacement of the Prometheus Pushgateway which allows for the pushing of
+    metrics to an endpoint for scraping by prometheus.
+
+containers:
+  prometheus-edge-hub:
+    resource: prometheus-edge-hub-image
+
+resources:
```

Comment:
Wait where is the metrics-endpoint provided relation?

Reply:
Just added it!